src/Component/MovieCard/MovieCards.js

```diff
@@ -2,9 +2,9 @@ import './MovieCards.css'
 
 
 function MovieCards(props){
-    const {movie} = props
+    const {movie, onClick} = props
```

Comment:
Could you make your function name more readable? Currently, it's named onCkick, which is passed to the onClick event listener.